pyproject.toml

```diff
@@ -90,16 +102,20 @@ pytest-env = "^1.1.3"
 testcontainers = "^4.8.1"
 pytest-timeout = "^2.3.1"
 
-[tool.poetry.group.test-scale.dependencies]
-locust = "^2.20.1"
-docker = "^7.0.0"
-matplotlib = "^3.8"
-pandas = "^2.2"
+# [tool.poetry.group.test-scale.dependencies]
+# locust = "^2.20.1"
+# docker = "^7.0.0"
+# matplotlib = "^3.8"
+# pandas = "^2.2"
```

Comment:
Should we remove this instead?

Reply:
I would like it keep it for now because we need to fix the scale tests altogether